src/braket/default_simulator/openqasm/program_context.py

```diff
@@ -890,3 +890,9 @@ def add_kraus_instruction(self, matrices: list[np.ndarray], target: list[int]):
 
     def add_result(self, result: Results) -> None:
         self._circuit.add_result(result)
+
+    def add_measure(self, target: int = None):
+        if target is None:
+            self._circuit.add_measure(list(range(self.num_qubits)))
```

Comment:
As above, I'm not sure when this case comes into play. Maybe this method should take a tuple though, since a measurement can be more than one qubit?

Reply:
Great point, I updated it to take a tuple instead.